api/regionProxy.go

```diff
@@ -30,6 +30,11 @@ func regionProxy(hostFormat, ownRegion string) hitch.Middleware {
 			respondError(rw, http.StatusLoopDetected, errors.New("proxy loop detected"))
 			return
 		}
+		// Clear any header we may have already set (e.g. CORS) to forward the
+		// response from the other region exactly as is.
+		for h := range rw.Header() {
```

Comment:
You sure? Nothing else at all relevant in there?

Reply:
The only header that we set in the response before this middleware (right now) are the CORS headers, so I think we are safe trimming them for now. Thinking about it, any other header that we may set here, should also be set on the upstream server when we do the region proxy, if it's running the same version. So I think it's relatively future-proof as well.

The only other alternative that I could think of would be to modify the headers on the `cors` middleware only before writing the headers to the response, i.e. after the reverse proxy has already started writing the body. But that would require "overriding" the `ResponseWritter.WriteHeaders` response to only write the headers then, and also have some issues with the other interfaces that the response writer implements like `http.Flusher`. So I think it's not really worth the complexity right now.